lib/modern_searchlogic/column_conditions.rb

```diff
@@ -233,8 +246,16 @@ def self.included(base)
         searchlogic_arel_alias :gte, :gteq
         searchlogic_arel_alias :less_than_or_equal_to, :lteq
         searchlogic_arel_alias :lte, :lteq
-        searchlogic_arel_alias :in, :in, :takes_array_args => true
-        searchlogic_arel_alias :not_in, :not_in, :takes_array_args => true
+        searchlogic_active_record_alias :in, :takes_array_args => true do |column, values|
+          where(column => values)
+        end
+        searchlogic_active_record_alias :not_in, :takes_array_args => true do |column, values|
+          if values.include?(nil)
+            query_not_in_with_nil(column,values)
+          else
+            where("#{column} NOT IN (?)", values)
+          end
```

Comment:
What about combining the above logic with `query_not_in_with_nil` so we don't have nil checking and `NOT IN (?)` building logic in both places.

Reply:
Sounds good, will update.